docs-v2/source/includes/_tipser-script.md

```diff
@@ -223,11 +223,15 @@ The complete list of HTML attributes related to modular product:
 
 Displays the full-size version of the currently selected product image.
 
+<aside class="notice">
+Product Image requires fixed width and height:
```

Comment:
Zmień to proszę na "the tag with `data-tipser-modular-product-image` attribute must have fixed width and height specified"